src/frontend/hooks/server/projects.ts

```diff
@@ -9,3 +10,13 @@ export function useProject() {
   const projectContext = useActiveProjectContext();
   return projectContext.project;
 }
+
+export function useProjectMembers() {
+  const project = useProject();
+  return useQuery({
+    queryFn: async () => {
+      return await project.$member.getMany();
+    },
+    queryKey: ['projectMembers'],
```

Comment:
I think we should add an optional [`refetchInterval`](https://tanstack.com/query/latest/docs/framework/react/reference/useQuery) to this hook, so that we can poll the backend to changes in this list. Project membership could change due to another device inviting a different device - that information would sync in the background with this device. I think updating this list of project members every 2-3 seconds would be ok. This is most relevant in training situations with many devices potentially inviting each other.